src/components/CoordinatorHome.js

```diff
@@ -4,149 +4,209 @@ import styled from "styled-components"
+                    {/* TODO: try changing to a radio button and making the result of the change equal
+                    to the opposite */}
+                    {/* onChange={e => store.setPhotoStatus('positive')} */}
+                    {/* store.current_strip_report = 'negative' */}
+                    <Input type="checkbox" name="photostatus" onChange={e => store.current_strip_report = 'positive'}
+                           checked={store.current_strip_report === 'positive'} /> Positive
+                    <br></br>
+                    <Input type="checkbox" name="photostatus" onChange={e => store.current_strip_report = 'negative'} 
+                          checked={store.current_strip_report === 'negative'} /> Negative / Unclear
+                    <br></br>
+                    {/* TODO: style like a button */}
+                    <Input type="submit" name="photostatus" onClick={e => (store.setPhotoStatus(store.current_strip_report))}/>
+                  </label>
+                </div>)
+        },
+        {
+          Header: "Contact",
+          accessor: "phone",
+          // Add's a What's App Link to the table
+          // TODO: make sure phone number is properly formatted in DB
+          Cell: e =><a href={'https://wa.me/' + e.value} target="_blank"> {e.value} </a>
+        },
+        {
+          Header: "Profile",
+          accessor: "id",
+          Cell: e=><a hrer={e.value} target="_black"> View </a>
+        },
+        {
+          Header: "My Notes",
+          accessor: "coordinator_note",
+          width: 175,
+          Cell: e=>
+          <div>
+            <Input use="textarea" />
+            <br></br>
+            <br></br>
+            <Button>Save Note</Button>
+          </div>
+          // TODO: Add a submit button
+        },
+      ]
+    },
+    {
+      Header: "Treatment",
+      columns: [
+        {
+          Header: "% Adherence",
+          accessor: "percent_since_start"
+        },      
+        {
+          Header: "Start Date",
+          accessor: "treatment_start_date"
+        },
+      ]
+    }
+  ];
+
+  return (
+    <Layout>
+      <span>{moment().format("YYYY-MM-DD")}</span>
+      <br />
+      <Heading>Manage Patient Progress</Heading>
+      <br />
+
+      <ReactTable
+            data={store.coordinator.patients}
+            columns={ tableColumns}
+            defaultPageSize={10}
+            showPagination={ false }
+            className="-striped -highlight"
+      />
+    </Layout>
+)})
+
+// STYLING TODO:
+// - get rid of the border
+// - what is the heading style?
 const Layout = styled.div`
   background-color: ${white};
   border: 1px solid ${darkgrey};
-  padding: 0.5rem;
 `
 
+// Padding not working on this styled element
 const Table = styled(ReactTable)`
   margin-top: 1rem;
+  width: 100%;
+  margin-left: 10px;
+  margin-right: 10px;
+  padding-top: 1rem;
+  padding-right: 1rem;
+  padding-bottom: 1rem;
+  padding-left: 1rem;
```

Comment:
I think `react-table`'s default styling is overriding everything here.
If you *need* to, you can add use `!important` to a CSS rule to get it to stick - but try to avoid it.

Reply:
Created a generic styling issue to cover this and the theming problems I was having with Notes.js